packages/react-next/src/components/Link/Link.types.ts

```diff
@@ -1,8 +1,11 @@
 import * as React from 'react';
+import { ComponentProps, ComposeOptions, ComposeStandardStatics } from '../../utils/tempTypes';
 import { IStyle, ITheme } from '../../Styling';
 import { IRefObject, IStyleFunctionOrObject } from '../../Utilities';
 import { IKeytipProps } from '../../Keytip';
 
+// tslint:disable:interface-name
```

Comment:
we should keep the I prefix until we have a clear story on how we plan to remove it